shell/common/persistent_cache.cc

```diff
@@ -4,10 +4,13 @@
 
 #include "flutter/shell/common/persistent_cache.h"
 
+#include <rapidjson/document.h>
```

Comment:
Nit: Square brackets for system headers and quotes for others. This goes at the bottom with the others.

Reply:
Done.